src/components/sections/Article.jsx

```diff
@@ -0,0 +1,62 @@
+import { Link, useParams } from "react-router-dom";
+import { useGetNewsQuery } from "../../features/newsSlice";
+import Wrapper from "../wrappers/Wrapper";
+
+const Article = () => {
+    const params = useParams();
+    const { articleId } = params;
+    console.log(articleId);
+    const { data, isSuccess } = useGetNewsQuery();
+    const articleData = isSuccess
+        ? data.find((article, index) => parseInt(articleId, 10) === index)
+        : null;
+    return (
+        <Wrapper>
+            {articleData !== null ? (
+                <div className="h-auto w-full flex justify-center items-center flex-col">
+                    <div className="md:w-1/2 my-5">
+                        <button className="bg-red-700 px-2 py-3 text-white font-semibold">
+                            <Link to="/">Go back to articles</Link>
+                        </button>
+                        <div className="">
+                            <h1 className="font-semibold text-2xl my-3">
+                                {articleData.title}
+                            </h1>
+                            <p className="font-bold text-gray-400 text-base">
+                                {" "}
+                                Published at: {articleData.publishedAt}
```

Comment:
I think you should format this date here like MM-DD-YY or YYYY/MM/DD or something similar

Reply:
got it, thanks